packages/tuff-cli-core/src/__tests__/builder-widgets.test.ts

```
@@ -156,6 +156,46 @@ describe('builder widget precompile', () => {
+      const manifest = await readManifest(root)
+      manifest.features[0].interaction = {
+        type: 'widget',
+        path: 'panel.arrow.ts',
+        runtime: 'arrow',
+      }
+      await writeManifest(root, manifest)
+      await fs.writeFile(
+        path.join(root, 'widgets', 'panel.arrow.ts'),
+        [
+          'import { component, html } from "@arrow-js/core"',
+          'export default component((props) => html`<section class="arrow-panel">${() => props.payload?.title || "Arrow"}</section>`)',
```

ESLint

[error] Unexpected template string expression.

(no-template-curly-in-string)